pages/_organism/_project.vue

```diff
@@ -457,13 +454,9 @@
         return window.location.href;
       },
     },
-    created() {
-      this.$store.commit('set_project_items', this.projectItems);
-    },
     mounted() {
       if (this.isError) return;
       this.checkSampleAlias();
-      this.$store.commit('set_project_filters', this.filters);
```

Comment:
avoid`set_project_filters` changing the `project_filters` to make the project results disappearing